hadoop-project/pom.xml

```diff
@@ -383,6 +383,13 @@
         <version>${hadoop.version}</version>
       </dependency>
 
+      <dependency>
```

Comment:
Goes away if the azure dependency change goes away?

Reply:
need to allow azure and gcs stores to subclass all the unit tests and uprate to integration tests. because, yes, gcs does it too.

Reply:
That can be a test only dependency? Dependency on the actual MR / YARN modules should not exist.